BaragonUI/app/components/groupDetail/util.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import _ from 'underscore';
+
+// Map a flat array into an array of arrays of size `size`, where the last
+// array is as long as possible
+// [1, 2, 3, 4, 5, 6, 7], 2 -> [[1, 2], [3, 4], [5, 6], [7]]
+// Precondition: size != 0
+function chunk(arr, size) {
```

Comment:
similar to the comment on the other PR, let's just keep a single utils file that can be shared everywhere